compose-cl.yml

```
@@ -0,0 +1,73 @@
+# Override any defaults specified by `${FOO:-bar}` in `.env` with `FOO=qux`.
+# ${VARIABLE:-default} evaluates to default if VARIABLE is unset or empty in the environment.
+# ${VARIABLE-default} evaluates to default only if VARIABLE is unset in the environment.
+
+services:
+  #                            _ _
+  #   __ _ _ __ __ _ _ __   __| (_)_ __   ___
+  #  / _` | '__/ _` | '_ \ / _` | | '_ \ / _ \
+  # | (_| | | | (_| | | | | (_| | | | | |  __/
+  #  \__, |_|  \__,_|_| |_|\__,_|_|_| |_|\___|
+  #  |___/
+
+  cl-grandine:
+    profiles: [cl_grandine]
+    image: sifrai/grandine:${GRANDINE_VERSION:-1.1.1}
+    restart: unless-stopped
+    command:
+      - --datadir=/root/.grandine
+      - --eth1-rpc-urls=http://el-${EL}:8551
```

```suggestion
      - --eth1-rpc-urls=http://el-${EL:-nethermind}:8551
```

Could we prevent user error with defaults like this actually?